packages/embeds/embed-react/package.json

```diff
@@ -23,7 +23,7 @@
     "withEmbedPublishEnv": "NEXT_PUBLIC_EMBED_LIB_URL='https://app.cal.com/embed/embed.js' NEXT_PUBLIC_WEBAPP_URL='https://app.cal.com' yarn",
     "prepack": "yarn ../../../ lint --filter='@calcom/embed-react' && yarn withEmbedPublishEnv build && yarn packaged:tests",
     "embed-web-start": "yarn workspace @calcom/web start",
-    "embed-dev": "yarn workspace @calcom/embed-react dev",
+    "embed-dev": "yarn workspace @calcoterm/embed-react dev",
```

Comment:
i doubt this should be calcoterm?

Reply:
Seems like a reasonable add-on to this fix.